src/pages-and-resources/app-settings-modal/AppSettingsModal.jsx

```diff
@@ -59,8 +61,18 @@ function AppSettingsModal({
 }) {
   const { courseId } = useContext(PagesAndResourcesContext);
   const loadingStatus = useSelector(getLoadingStatus);
```

Comment:
@mehaknasir as discussed we are going to use ModelDialog here so most of the code here will be updated. Do you think this PR should also update / change something which eventually be upgraded?

Reply:
I think no. let's keep it that way unless we have the paragon component up and running according to our requirement.

Reply:
thanks.